Description: Fix git over HTTPS
Origin: upstream, commits 1a02e84 and 2045d83
Bug-Debian: http://bugs.debian.org/690551
Author: Alessandro Ghedini <ghedo@debian.org>
Reviewed-by: Alessandro Ghedini <ghedo@debian.org>
Last-Update: 2012-11-16

diff -u b/lib/gtls.c b/lib/gtls.c
--- b/lib/gtls.c
+++ b/lib/gtls.c
@@ -299,10 +299,11 @@
       connssl->connecting_state =
         gnutls_record_get_direction(session)?
         ssl_connect_2_writing:ssl_connect_2_reading;
+      continue;
       if(nonblocking)
         return CURLE_OK;
     }
-    else if((rc < 0) && gnutls_error_is_fatal(rc)) {
+    else if((rc < 0) && !gnutls_error_is_fatal(rc)) {
       failf(data, "gnutls_handshake() warning: %s", gnutls_strerror(rc));
     }
     else if(rc < 0) {
