Description: Fix git authentication when HTTPS is used
Origin: upstream, commits da82f59 and 8d97bed
Bug-Debian: http://bugs.debian.org/690764
Author: Alessandro Ghedini <ghedo@debian.org>
Reviewed-by: Alessandro Ghedini <ghedo@debian.org>
Last-Update: 2012-11-16

diff -u b/lib/transfer.c b/lib/transfer.c
--- b/lib/transfer.c
+++ b/lib/transfer.c
@@ -1433,10 +1433,6 @@
 
   data->state.ssl_connect_retry = FALSE;
 
-  /* zero out auth state */
-  memset(&data->state.authhost, 0, sizeof(struct auth));
-  memset(&data->state.authproxy, 0, sizeof(struct auth));
-
   data->state.authproblem = FALSE;
   data->state.authhost.want = data->set.httpauth;
   data->state.authproxy.want = data->set.proxyauth;
@@ -1473,6 +1469,12 @@
 
     if(data->set.connecttimeout)
       Curl_expire(data, data->set.connecttimeout);
+
+    /* In case the handle is re-used and an authentication method was picked
+       in the session we need to make sure we only use the one(s) we now
+       consider to be fine */
+    data->state.authhost.picked &= data->state.authhost.want;
+    data->state.authproxy.picked &= data->state.authproxy.want;
   }
 
   return res;
