Description: Fix running curl binary in tests
Origin: upstream
Forwarded: not-needed
Author: Daniel Stenberg <daniel@haxx.se>
Reviewed-by: Alessandro Ghedini <ghedo@debian.org>
Last-Update: 2018-10-31

--- a/tests/runtests.pl
+++ b/tests/runtests.pl
@@ -151,7 +151,7 @@
 
 my $srcdir = $ENV{'srcdir'} || '.';
 my $CURL="../src/curl".exe_ext(); # what curl executable to run on the tests
-my $VCURL="curl";   # what curl binary to use to verify the servers with
+my $VCURL=$CURL;   # what curl binary to use to verify the servers with
                    # VCURL is handy to set to the system one when the one you
                    # just built hangs or crashes and thus prevent verification
 my $DBGCURL=$CURL; #"../src/.libs/curl";  # alternative for debugging
